packages/button/README.md

```diff
@@ -0,0 +1,22 @@
+# React Material Icon
+
+MDC React Material Icon is a component for material icons.
+
+## Installation
+
+```
+npm install @material/react-button
+```
+
+## Props
+
+Prop Name | Type | Description
+--- | --- | ---
+className | String | Classes to be applied to the root element.
+raised | n/a | Enables raised variant.
+unelevated | n/a | Enables unelevated variant.
+stroked | n/a | Enables stroked variant.
+dense | n/a | Enables dense variant.
+icon | Element | Icon to render within root element.
```

Comment:
I think the icon should be the children, since it is an element

Reply:
It is nonintuitive - I don't think being an element should dictate if its `children` or through a `prop`. We should go with what makes sense.

I like switching to `textLabel` and `icon` as props. Its more declarative. What do you think?